File: Modul 4/C/PRAK405-23108317310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

int main(){
    int x, y, i, j;
    scanf("%d %d",&x, &y);

    int hasil = 0;

    for (i = 1; i <= x; i++)
    {
        int total = 0;
        for (j = i; j >= 1; j--)
        {
            int perkalian_kelipatan = j * y;
            printf("(%d * %d)", j, y);
            total += perkalian_kelipatan;
            if (j > 1)
            {
                printf(" + ");
            }
            
        }
        printf(" = %d\n", total);
        hasil += total; 
    
    }

    printf("%d\n", hasil);

    return 0;
}

File: Modul 4/Python/PRAK405-23108317310005-MuhammadNurwahyudiAdhitama.py
x, y = map(int, input("").split())

hasil = 0

for i in range(1, x + 1):
    total = 0
    for j in range(i, 0, -1):
        perkalian_kelipatan = j * y
        print(f"({j} * {y})", end="")
        total += perkalian_kelipatan
        if j > 1:
            print(" + ", end="")
    
    print(f" = {total}")
    hasil += total

print(hasil)
